src/transaction/transaction_master_group_complete_manager.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+//
+// Manager of completed group
+//
+
+// todo: use cubtx namespace
+
+#ifndef _MASTER_GROUP_COMPLETE_MANAGER_HPP_
+#define _MASTER_GROUP_COMPLETE_MANAGER_HPP_
+
+#include "transaction_group_complete_manager.hpp"
+#include "stream_transfer_sender.hpp"
+
+#include "cubstream.hpp"
+#include "thread_entry_task.hpp"
+#include "thread_manager.hpp"
```

Comment:
thread_manager.hpp seems unnecessary; it may be replaced with thread_daemon.hpp

header should be ordered alphabetically.